completion/available/flutter.completion.bash

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/bash
+
+if command -v flutter > /dev/null; then
+    eval "$(flutter bash-completion)"
```

Comment:
This should be indented with two spaces - please take a look at the `.editorconfig` file in the project root.

Reply:
I am sorry sir its my IDE :man_facepalming: